src/global/trampoline.rs

```diff
@@ -354,9 +354,14 @@ impl Trampoline {
             .into_diagnostic()?;
         }
 
-        // Create a hard link to the shared trampoline binary
-        if !self.path().exists() {
-            tokio::fs::hard_link(self.trampoline_path(), self.path())
+        // If the path doesn't exist yet, create a hard link to the shared trampoline binary
+        // If creating a hard link doesn't succeed, try copying
```

Comment:
could we expand here the comment explaining for what specific use-case we are trying to fallback to copying?

Reply:
I mean there was one specific issue, but in general the use-case is to fall-back if hard-linking doesn't work :)

Reply:
I think it would be nice to mention that there could be a limit for nr of hardlinks so we are trying to fallback because of this, so for a new contributor it would be easier to understand why because `std::hard_link` is not super verbose about this kind of errors

Reply:
Fair enough, extended the comment